modules/retail/pages/retail-deploy-terminals-other.adoc

```diff
@@ -0,0 +1,173 @@
+====
+
+
+
+== Deploy Terminals with a Removable USB Device
+
+If you do not want to boot terminals from the network, you can create a live USB image and deploy terminals using a removable USB storage device.
+This is useful if you cannot boot your terminals from the network, or if you do not have a local {smr} branch server providing network services.
+
+You can prepare a bootable USB device with the image and tools required to deploy a POS terminal using a remote {smr} branch server.
+POS devices booted using the USB device are assigned to the {smr} branch server that created the USB device.
+
+You can create the bootable USB device on the branch server directly, or on the {smr} Server.
+
+
+
+.Procedure: Creating a Bootable USB Device
+. On the {smr} branch server, at the command prompt, as root, create the POS image.
+You need to specify the size of the image, in megabytes.
+Ensure you allow at least 300{nbsp}MB:
++
+----
+salt-call image_sync_usb.create <usb image name> <size in MB>
+----
+. Insert the USB device into the {smr} branch server machine, and copy the image to the new location:
++
+----
+dd if=<usb image name> of=<path to usb device>
+----
+
+
+When you have the image on the USB drive, check that the terminals you want to deploy allow local booting.
+You can check this by editing the Saltboot formula in the {smr} {webui}.
+For more information about the Saltboot formula, see xref:salt:formula-saltboot.adoc[].
+
+
+
+.Procedure: Deploying Images to the Terminals using USB
+. Insert the USB device into the terminal.
+. Power on the POS terminal.
+. Boot from the USB device to begin bootstrapping.
+
+
+
+== Bootstrap Terminals over a Wireless Network
+
+For terminals that cannot be connected directly to the physical network, you can bootstrap them over a wireless network.
+Wireless networks do not support PXE booting, so you must perform the initial booting and initialization of the wireless connection on the terminal using a USB device.
+
+For more information about using USB devices to boot, see xref:retail:retail-deploy-terminals-other.adoc[].
+
+
+[[WARNING]]
+====
+Bootstrapping across a wireless network could expose a security risk if you are using encrypted OS images.
+The boot ``initrd`` image and the partition that contains ``/etc/salt`` must be stored unencrypted on the terminal.
+This allows {smr} to set up the wireless network.
+If this breaches your security requirements, you will need to use a separate production network, with network credentials managed by Salt, so that credentials are not stored on the terminal unencrypted.
+====
+
+
+.Procedure: Preparing Wireless-Enabled Boot Image
+. Ensure that the ``dracut-wireless`` package is included  in the image template.
+. Provide the wireless credentials using the [command]``WIRELESS_ESSID`` and [command]``WIRELESS_WPA_PSK`` kernel options.
```

Comment:
```suggestion
```

This line really shouldn't be here.

For the wifi enabled image the only mandatory part is that it must have `dracut-wireless` package included (line above this one)
Then optional part is the line below this, adding wifi credentials to the image itself using files.
This line about kernel options is not relevant to the image building, only when terminal is booting up way after image has been built.